components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2483,7 +2483,8 @@ private Response handleAuthFlowThroughFramework(OAuthMessage oAuthMessage, Strin
                 if (attribute == AuthenticatorFlowStatus.INCOMPLETE) {
 
                     if (responseWrapper.isRedirect()) {
-                        oAuthMessage.getResponse().sendRedirect(responseWrapper.getRedirectURL());
+                        return Response.status(HttpServletResponse.SC_FOUND).location(
```

Comment:
What's the reason for this change?

Reply:
@ashendes , did we check removing this.
The first explation was this was added as this "returns 204 instead of 302 when servlet forwards are done at the rest dispatcher"